### ISSUE TITLE
Question about test episodes

### ISSUE THREAD

**Issue by JackieLiu0414** [Jia-Research-Lab/PFENet#23]
Thank you for sharing! I'm confused about the set where test episodes are sampled. In this work, datasets are divided into training set and validation set. As far as I'm concerned, both train and val sets contain 4 folds. If you test on fold-0, training episode of fold-1,2,3 are sampled from training set, and, test episodes of fold-0 are sampled from val set?  Do I have any misunderstandings? Thank you.

**Comment by tianzhuotao** [comment]
@JackieLiu0414 

Yes, your understanding is correct. Following previous works, we perform cross-validation on all datasets that are split into four splits. When testing on one split, the other three splits are used for training.

More details can be found in the dataset.py

**Comment by JackieLiu0414** [comment]
Thanks again.
